automation/taskcluster/action/update_geckoview.sh

```diff
@@ -61,6 +68,11 @@ else
     exit 1
 fi
 
+# The build system automatically updates the docs. Add them to the
+# docs PR.
+git add components/browser/${COMPONENT_NAME}/docs/metrics.md
```

Comment:
`git add` seems to fail with a non-zero exit code if the file does not exist. This will cause this script to fail. We need to handle this case for browser-engine-gecko(-*) flavors that do not have that file yet.

Reply:
Great catch.